src/simulation/SimulationManager.java

```diff
@@ -53,12 +54,21 @@ public SimulationManager() {
         constraintManager = new ConstraintManager();
 
         traceMode = false;
+        diffMode = false;
     }
 
     public boolean isTrace() {
         return traceMode;
     }
 
+    public void setDiffMode(boolean b) {
```

Comment:
I'm okay with this approach for now, but long term (if we add more settings) we really should have a `SettingsManager` that maintains keys and values in a map.